test/pom.xml

```diff
@@ -162,6 +162,19 @@
 
 	<build>
 		<defaultGoal>package</defaultGoal>
+
+		<!-- required for the Cypress tests, both the default, and the new path, else it won't use the default test-resources -->
+		<!-- Since intellij is dumb af and uses absolute paths, use iaf.rootdir -->
+		<testResources>
+			<testResource>
+				<directory>src/test/resources</directory>
+			</testResource>
+			<testResource>
+				<directory>${iaf.rootdir}/console/frontend/src/main/frontend</directory>
```

Comment:
This broke the build in IntelliJ and in the CI

Reply:
@nielsm5 could you look at this again?

Reply:
Reverted the changes, and added a comment

Reply:
Changed entirely